<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>587</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>奇路小说阅读器 V0.1</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>0</y>
      <width>16</width>
      <height>141</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="textOut" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>882</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuCodec">
    <property name="title">
     <string>编码</string>
    </property>
    <addaction name="actionGBK"/>
    <addaction name="actionUnicode"/>
    <addaction name="actionUnicode_Big_Endian"/>
    <addaction name="actionUTF_8"/>
    <addaction name="actionBIG5"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCodec"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionGBK">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GBK</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>选项...</string>
   </property>
  </action>
  <action name="actionUnicode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Unicode</string>
   </property>
  </action>
  <action name="actionUTF_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
  <action name="actionBIG5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BIG5</string>
   </property>
  </action>
  <action name="actionUnicode_Big_Endian">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Unicode Big Endian</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>textOut</class>
   <extends>QWidget</extends>
   <header>textout.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
